src/hotspot/share/gc/g1/heapRegion.cpp

```diff
@@ -798,7 +798,7 @@ void HeapRegion::mangle_unused_area() {
 #endif
 
 void HeapRegion::initialize_bot_threshold() {
-  _bot_part.initialize_threshold();
+  _bot_part.reset_bot();
```

Comment:
As a follow-up it might be nice to also rename the function here and update the usage to have matching naming for parameters and such.

Reply:
Maybe sth similar to https://bugs.openjdk.java.net/browse/JDK-8279008 can be done for `G1BlockOffsetTablePart::_next_offset_threshold` as well. Then, there's no need to (re)init the threshold.